activities/FoodChain.activity/index.html

```diff
@@ -36,6 +36,13 @@
 	<script src="app.js" type="text/javascript"></script>
     <script>if (window.module) module = window.module;</script>
 	<link href="styles.css" rel="stylesheet" type="text/css" />
+	<!-- Tutorial -->
+	<link rel="stylesheet" href="bootstrap-tour-standalone.min.css">
+	<script>if (typeof module === 'object') {window.module = module; module = undefined;}</script>
+	<script src="lib/jquery.min.js"></script>
+	<script src="lib/bootstrap-tour-standalone.min.js"></script>
+	<!-- <script>if (window.module) module = window.module;</script> This line already exists--> 
```

Comment:
Should not be commented, inclusion of JS libraries should be surrounded by the two lines mentioned.